roboconf-dist-rpm-agent/pom.xml

```diff
@@ -0,0 +1,279 @@
+						<goals>
+							<goal>copy-dependencies</goal>
+						</goals>
+						<configuration>
+							<overWriteReleases>false</overWriteReleases>
+							<overWriteSnapshots>false</overWriteSnapshots>
+							<overWriteIfNewer>true</overWriteIfNewer>
+							<excludeTransitive>true</excludeTransitive>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<!-- unpack dependencies -->
+<!-- 			<plugin> -->
+<!-- 				<artifactId>maven-dependency-plugin</artifactId> -->
+<!-- 				<executions> -->
+<!-- 					<execution> -->
+<!-- 						<phase>package</phase> -->
+<!-- 						<id>unpack-dependencies</id> -->
+<!-- 						<goals> -->
+<!-- 							<goal>unpack-dependencies</goal> -->
+<!-- 						</goals> -->
+<!-- 						<configuration> -->
+<!-- 							<overWriteReleases>false</overWriteReleases> -->
+<!-- 							<overWriteSnapshots>false</overWriteSnapshots> -->
+<!-- 							<overWriteIfNewer>true</overWriteIfNewer> -->
+<!-- 							<excludeTransitive>true</excludeTransitive> -->
+<!-- 							<outputDirectory>${project.build.directory}/dependency</outputDirectory> -->
+<!-- 						</configuration> -->
+<!-- 					</execution> -->
+<!-- 				</executions> -->
+<!-- 			</plugin> -->
+			
+
+			<!-- Then, generate the RPM package -->
+			<plugin>
+			    <groupId>org.codehaus.mojo</groupId> 
+				<artifactId>rpm-maven-plugin</artifactId>
+				<version>2.1.4</version> 
+				<executions>
+					<execution>
+						<id>generate-rpm</id>
+						<goals>
+							<goal>rpm</goal>
+						</goals>						
+						<configuration>
+						    <group>Applications/Internet</group>
+						    <mappings>
+						    <!-- Karaf distribution is located under /opt/roboconf-agent -->
+						    	<mapping>
+						    		<directory>/opt/roboconf-agent</directory>
+						    		<directoryIncluded>false</directoryIncluded>
+						    		<username>roboconf-agent</username>
+						    		<groupname>roboconf-agent</groupname>
+						    		<filemode>755</filemode>
+						    		<sources>
+						    			<source>
+						    				<location>${project.build.directory}/dependency/roboconf-karaf-dist-agent-${project.parent.version}.tar.gz</location>
+<!-- 						    				<destination>roboconf-karaf-dist-agent-0.6-SNAPSHOT.tar.gz</destination> -->
+						    			</source>
+						    		</sources>
+						    	</mapping>
+						    	
+						    <!--Copy the init.d script-->	
+						    	<mapping>
+						    		<directory>/etc/init.d</directory>
+						    		<directoryIncluded>false</directoryIncluded>
+						    		<username>root</username>
+						    		<groupname>root</groupname>
+						    		<filemode>755</filemode>
+						    		<sources>
+						    			<source>
+						    				<location>${project.basedir}/src/files/bin/roboconf-agent-init.d</location>
+						    				<destination>roboconf-agent</destination>
+						    			</source>
+						    		</sources>
+						    	</mapping>
+						    	
+						    <!--Copy the log configuration file-->	
+<!-- 						    	<mapping> -->
```

Comment:
Sure. But are these files copied at the right location?
Your scripts set the KARAF_ETC variable to /etc/roboconf-agent (resp. -dm). So, logger configuration must be placed there too.